module/VuFind/src/VuFind/RecordDriver/SolrMarcRemote.php

```diff
@@ -32,7 +32,7 @@
 
 namespace VuFind\RecordDriver;
 
-use Laminas\Log\LoggerAwareInterface as LoggerAwareInterface;
+use VuFind\Log\LoggerAwareInterface as LoggerAwareInterface;
```

Comment:
This can be simplified:

```suggestion
use VuFind\Log\LoggerAwareInterface;
```